model/version.go

```diff
@@ -19,4 +19,10 @@ type VersionInfo struct {
 	// required: true
 	// example: 2018-02-27T19:36:10.5045044+01:00
 	BuildDate string `json:"buildDate"`
+
+	// Registration is allowed
+	//
+	// required: false
+	// example: true
+	Registration bool `json:"registration"`
```

Comment:
I dislike this, as registration has really nothing to do with the version endpoint. There is probably a better solution by injecting the into the HTML at runtime, I'll do this after the PR.